modes/elisp-mode/lem-elisp-mode.asd

```diff
@@ -2,6 +2,7 @@
   :depends-on ("lem"
                "jsonrpc"
                "lem-lisp-mode"
+               "jsonrpc/transport/http"
```

Comment:
@Sasanidas 
I'm sorry.
After all, this dependence on ironclad, crack, etc. seems to be quite large.
Is there any way we can delay it until we use a specific feature of elisp-mode?

Reply:
Umm I see, I can separate the elisp-mode form the rpc interface, so elisp-mode will only provide basic major-mode

Reply:
I think this should solve the issue https://github.com/lem-project/lem/pull/825 :+1: